modules/nodes-nodes-kernel-arguments.adoc

```diff
@@ -162,11 +162,16 @@ Starting pod/ip-10-0-141-105ec2internal-debug ...
 To use host binaries, run `chroot /host`
 
 sh-4.2# cat /host/proc/cmdline
-BOOT_IMAGE=/ostree/rhcos-... console=tty0 console=ttyS0,115200n8
+BOOT_IMAGE=/ostree/rhcos-... ,115200n8
 rootflags=defaults,prjquota rw root=UUID=fd0... ostree=/ostree/boot.0/rhcos/16...
 coreos.oem.id=qemu coreos.oem.id=ec2 ignition.platform.id=ec2 selinux=0
 
 sh-4.2# exit
 ----
 +
 You should see the `selinux=0` argument added to the other kernel arguments.
++
+[NOTE]
+====
+To access serial console, append the correct `console=` setting for your hardware platform. Most users can use `console=ttyS0`. This configuration does not enable serial console access on machines with a graphical console.  To configure a different console, add one or more `console=` arguments to the `APPEND` line.  For example, add `console=tty0 console=ttyS0` to set the first PC serial port as the primary console and the graphical console as a secondary console.  For more information, see https://access.redhat.com/articles/7212[this article].
```

Comment:
see link:https://access.redhat.com/articles/7212[How does one set up a serial terminal and/or console in Red Hat Enterprise Linux?].

Reply:
...add one or more `console=` arguments to the `kernelArguments` section.

Reply:
Whoops, https://github.com/openshift/openshift-docs/pull/29525#discussion_r578793949 is correct.  Disregard my comment.